src/P3.jl

```diff
@@ -1,15 +1,18 @@
 """
-Predicted particle properties scheme P3 for ice.
-Implementation of Morrison and Milbrandt 2015 doi: 10.1175/JAS-D-14-0065.1
+Predicted particle properties scheme P3 for ice [MorrisonMilbrandt2015](@cite).
 
-Note: Particle size is defined as its maximum length (i.e. max dimesion).
+!!! note "Particle size definition" 
```

Comment:
This part shows in the API docs, but I think it's not needed there. Maybe we can just move it to the documentation page.